stable/nginx-ingress/README.md

```diff
@@ -138,6 +138,10 @@ Parameter | Description | Default
 `controller.metrics.serviceMonitor.additionalLabels` | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus | `{}`
 `controller.metrics.serviceMonitor.namespace` | namespace where servicemonitor resource should be created | `the same namespace as nginx ingress`
 `controller.metrics.serviceMonitor.honorLabels` | honorLabels chooses the metric's labels on collisions with target labels. | `false`
+`controller.metrics.prometheusRule.enabled` | Set this to `true` to create prometheusRules for Prometheus operator | `false`
+`controller.metrics.prometheusRule.additionalLabels` | Additional labels that can be used so prometheusRules will be discovered by Prometheus | `{}`
+`controller.metrics.prometheusRule.namespace` | namespace where prometheusRules resource should be created | `the same namespace as nginx ingress`
```

Comment:
namespace is handled by helm and doesn't need to be included in our templates.

Reply:
Yes, I understand namespace is handled by helm but for most of the situation you will require to specify another one different and this gives you the possibility. Actually, it's my case: we have the monitoring/alerting system in another namespace different than the one where nginx-ingress is deployed. 

You're already implementing the same approach in this chart, check `templates/controller-servicemonitor.yaml` ( https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/controller-servicemonitor.yaml#L6-L8 )